server/data/Rajagopal2015/raw/_subject.json

```diff
@@ -8,7 +8,6 @@
   "email": "nbianco@stanford.edu",
   "skeletonPreset": "custom",
   "disableDynamics": false,
-  "segmentTrials": true,
```

Comment:
Everything looks good. I thing it could be good if there were an example of "segmentedTrials" here, as there was "segmentTrials" before.

Reply:
Done.